src/pages/buy-sell/components/SortDropdown/SortDropdown.tsx

```diff
@@ -32,7 +33,6 @@ const SortDropdown = ({ list, onSelect, setIsFilterModalOpen, value }: TSortDrop
     return (
         <div className='sort-dropdown' data-testid='dt_sort_dropdown'>
             <Dropdown
-                chevronIcon={<LabelPairedChevronDownMdRegularIcon />}
```

Comment:
why are we removing the icon?

Reply:
ah i need to put it back 😅 but in the dropdown component they have it by default, but ill put it back